solutions/usaco-839.mdx

```diff
@@ -2,58 +2,87 @@
 id: usaco-839
 source: USACO Gold 2018 Open
 title: Talent Show
-author: Arpan Banerjee
+author: Jesse Choe
 ---
 
```

Comment:
what's the point of changing this to a solution that's super similar to the editorial and replacing the other solution?

Reply:
fair point, but the misconception is lowkey kinda cringe 😛